komorebi-bar/src/bar.rs

```diff
@@ -50,6 +53,7 @@ pub struct Komobar {
     pub render_config: Rc<RefCell<RenderConfig>>,
     pub komorebi_notification_state: Option<Rc<RefCell<KomorebiNotificationState>>>,
     pub left_widgets: Vec<Box<dyn BarWidget>>,
+    pub center_widgets: Vec<Box<dyn BarWidget>>,
```

Comment:
I think we can probably bring this into `master` after making `center_widgets` an `Option<T>` so that users with existing configs won't get this error on upgrade:

```
❯ komorebi-bar
2024-11-25T23:39:20.411510Z  INFO komorebi_bar: found configuration file: C:\Users\LGUG2Z\.config\komorebi\komorebi.bar.json
Error:
   0: missing field `center_widgets` at line 95 column 1

Location:
   komorebi-bar\src\config.rs:103

```

Reply:
@LGUG2Z I've made the config optional and fixed the spacing issue as well. I think it's good to go.